src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Generated/Models/QueryFailureError.cs

```diff
@@ -45,17 +45,17 @@ public QueryFailureError(string code = default(string), string message = default
         partial void CustomInit();
 
         /// <summary>
-        /// Gets or sets service specific error code which serves as the
-        /// substatus for the HTTP error code.
+        /// Gets service specific error code which serves as the substatus for
+        /// the HTTP error code.
         /// </summary>
         [JsonProperty(PropertyName = "code")]
-        public string Code { get; set; }
+        public string Code { get; private set; }
 
         /// <summary>
-        /// Gets or sets description of the error.
+        /// Gets description of the error.
         /// </summary>
         [JsonProperty(PropertyName = "message")]
-        public string Message { get; set; }
+        public string Message { get; private set; }
```

Comment:
This is a breaking change for existing customers, please bump major version number

Reply:
Done.  This model is a response model only used for error responses so it should be very unlikely that someone was setting the properties that are now correctly marked as readonly.

Reply:
Agreed, I'd take a second opinion from @shahabhijeet before approving